README.md

```diff
@@ -26,6 +26,9 @@ And if you spot bugs or have features that you'd really like to see in `gh`, ple
 
 Check out the [docs][] for more information.
 
+## Documentation
+
+Read the official docs at [https://cli.github.com/manual/](https://cli.github.com/manual/)
```

Comment:
I feel like this should be explicitly called out at the the beginning of the `README`. I personally didn't know this existed.

Reply:
Have you seen the line the precedes the section that you've added? I feel like this is just repeating information at this point.

Reply:
Whoops! Sorry, I actually did not that see that. You're right though. It does feel repetitive. What if we deleted that line before and added this new section?

Reply:
I'm into this.